sky/templates/kubernetes-sshjump.yml.j2

```diff
@@ -0,0 +1,83 @@
+pod_spec:
+    apiVersion: v1
+    kind: Pod
+    metadata:
+        name: {{ name }}
+        labels:
+            component: {{ name }}
+    spec:
+        serviceAccountName: sshjump
+        volumes:
+          - name: secret-volume
+            secret:
+                secretName: {{ secret }}
+        containers:
+          - name: {{ name }}
+            imagePullPolicy: IfNotPresent
+            image: {{ image }}
+            command: ["python3", "-u", "/skypilot/sky/utils/kubernetes/sshjump-lcm.py"]
+            ports:
+              - containerPort: 22
+            volumeMounts:
+              - name: secret-volume
+                readOnly: true
+                mountPath: /etc/secret-volume
+            lifecycle:
+                postStart:
+                    exec:
+                        command: ["/bin/bash", "-c", "mkdir -p ~/.ssh && cp /etc/secret-volume/ssh-publickey ~/.ssh/authorized_keys && sudo service ssh restart"]
+            env:
+              - name: MY_POD_NAME
+                valueFrom:
+                    fieldRef:
+                        fieldPath: metadata.name
+              - name: MY_POD_NAMESPACE
+                valueFrom:
+                    fieldRef:
+                        fieldPath: metadata.namespace
+              - name: ALERT_THRESHOLD
+                # seconds
+                value: "600"
+              - name: RETRY_INTERVAL
+                # seconds
+                value: "60"
+        terminationGracePeriodSeconds: 0
+service_spec:
+    apiVersion: v1
+    kind: Service
+    metadata:
+        name: {{ name }}
+    spec:
+        type: NodePort
+        selector:
+            component: {{ name }}
+        ports:
+          - protocol: TCP
+            port: 22
+            targetPort: 22
+service_account:

```

Comment:
I think it'd be nice to add a comment here that the following `ServiceAccount`/`Role`/`RoleBinding` sets up an RBAC for life cycle management of the jump pod/service.